.github/workflows/actions.yml

```diff
@@ -304,13 +304,182 @@ jobs:
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: prepare timestamp for cache
+        id: current-date
+        run: |
+          echo "::set-output name=stamp::$(date '+%Y-%m-%d')"
+          echo "::set-output name=week::$(date '+%V')"
+      - name: cache qt
+        id: cache-qt
+        uses: actions/cache@v1
+        with:
+          path: ../Qt
+          key: ${{ runner.os }}-v1-${{ matrix.os-version }}-${{ matrix.qt-version }}-qt${{ matrix.qt-arch }}
+      - name: install qt using aqtinstall
+        uses: jurplel/install-qt-action@v2
+        with:
+          modules: 'qtwebengine'
+          version: ${{ matrix.qt-version }}
+          arch: ${{ matrix.qt-arch }}
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+      - name: install libsndfile
+        run: choco install libsndfile ${{ matrix.chocolatey-options }}
+      - name: download fftw
+        shell: bash
+        env:
+          FFTW_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/fftw
+        run: |
+          mkdir -p $FFTW_PATH && cd $FFTW_PATH
+          curl -L ${{ matrix.fftw-url }} -o fftw.zip
+          7z x fftw.zip -y
+          # add env var for subsequent steps
+          echo 'FFTW_PATH<<EOF' >> $GITHUB_ENV
+          echo $FFTW_PATH >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: create fftw msvc library
+        if: env.VCVARS_SCRIPT
+        shell: cmd
+        env:
+          FFTW_ARCH: ${{ matrix.fftw-arch }}
+        run: |
+          call "%VCVARS_SCRIPT_PATH%"
+          cd "%FFTW_PATH%"
+          lib.exe /machine:%FFTW_ARCH% /def:libfftw3f-3.def
+      - name: move fftw
+        shell: bash
+        run: |
+          mv $FFTW_PATH "C:/Program Files/fftw"
+          echo "Done installing fftw"
+      - name: install asio sdk
+        shell: bash
+        env:
+          ASIO_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/asio_sdk
+        run: |
+          mkdir -p $ASIO_PATH && cd $ASIO_PATH
+          curl -L $ASIO_URL -o asio.zip
+          7z x asio.zip -y
+          mv $ASIO_FOLDER_NAME $GITHUB_WORKSPACE/external_libraries/portaudio/asiosdk
+          echo "Done installing asio sdk"
+      - name: configure
+        shell: bash
+        env:
+          CMAKE_GENERATOR: ${{ matrix.cmake-generator }}
+          CMAKE_ARCH: ${{ matrix.cmake-arch }}
+          USE_WEBENGINE: ${{ matrix.use-qtwebengine }}
+          ABLETON_LINK: ${{ matrix.ableton-link }}
+        run: |
+          export PATH=$Qt5_DIR/bin:$PATH
+
+          mkdir $BUILD_PATH && cd $BUILD_PATH
+
+          cmake -G "$CMAKE_GENERATOR" -A "$CMAKE_ARCH" -D CMAKE_PREFIX_PATH="$Qt5_DIR" -D SUPERNOVA=ON -D SC_USE_QTWEBENGINE=$USE_WEBENGINE -D SC_ABLETON_LINK=$ABLETON_LINK -D CMAKE_BUILD_TYPE=Release .. # build type is specified here for MinGW build
+
+      - name: build
+        shell: bash
+        run: |
+          cmake --build $BUILD_PATH --config Release --target install
```

Comment:
for minGW, since it's using make, you can add `-- -j2` at the end of this command to take advantage of both cores on the build machines. i would call that `BUILD_PARALLELISM`

Reply:
okay, this prompted me to properly set the build config for MSVC vs MinGW - now I'm setting that in Bash based on the values from the matrix. Is that okay?
https://github.com/supercollider/supercollider/pull/5377/files#diff-2702f6f7d10a1bffbc8596d1a9925b1234bc4c7a6ff4c21ca6b34c13aa75780eR444
https://github.com/supercollider/supercollider/pull/5377/files#diff-2702f6f7d10a1bffbc8596d1a9925b1234bc4c7a6ff4c21ca6b34c13aa75780eR453